File: C/2286 - Par ou Ímpar.c
#include <stdio.h>

int main()
{
    int n, teste=1;
    
    scanf("%d",&n);
    while (n!=0){
        
        char jog1[11],jog2[11];
        scanf(" %s",&jog1);
        scanf(" %s",&jog2);
        printf("Teste %d\n",teste);
        for (int c=0; c<n; c++){
            int A,B,soma;
            scanf("%d %d",&A,&B);
            
            soma= A+B;
            if (soma%2==0){
                printf("%s\n",jog1);
            }
            else{
                printf("%s\n",jog2);
            }
        }
        printf("\n");
        teste++;
        scanf("%d",&n);
        
    }

   

    return 0;
}

File: Python/2286 - Par ou Ímpar.py
teste = 1
while True:
    N = int(input())
    if N == 0:
        break
    
    jogador1 = input().strip()
    jogador2 = input().strip()
    
    print(f"Teste {teste}")
    for _ in range(N):
        A, B = map(int, input().split())
        soma = A + B
        if soma % 2 == 0:
            print(jogador1)
        else:
            print(jogador2)
    
    print()  
    teste += 1
